analyzer/emerald/incoming.go

```diff
@@ -400,7 +432,311 @@ func extractRound(b *block.Block, txrs []*sdkClient.TransactionWithResults, logg
 	return &blockData, nil
 }
 
-func emitRoundBatch(batch *storage.QueryBatch, qf *analyzer.QueryFactory, round uint64, blockData *BlockData) {
+var ABIERC20 abi.ABI
+
+func init() {
+	var err error
+	ABIERC20, err = abi.JSON(strings.NewReader(`
+  [
+    {
+      "constant": true,
```

Comment:
might be nice to have this compacted to be one line per abi entry. a middle ground between this menace to scrolling and an impenetrable single uglified line

Reply:
that file doesn't get shipped with the compiled indexer program, and I don't want us to use "our docker build will still work" as an excuse to complicate the indexer program.

let's look for ways to embed the file at build time

Reply:
o crud I forgot that the artifact json is a larger object with a .abi property

Reply:
ok there's an easy enough way to say that we want just the .abi part, although we end up embedding the whole artifact json.

Reply:
changed to load from ERC20.json. at compile time